src/silverlabnwb/nwb_file.py

```diff
@@ -369,11 +370,19 @@ def add_time_series_data(self, label, data, times, ts_attrs={}, data_attrs={},
         :param ts_attrs: Any attributes for the timeseries group itself.
         :param data_attrs: Any attributes for the data array.
         :param kind: The class of timeseries to create, e.g. TwoPhotonSeries.
+        :param compress: True if data should be compressed, False otherwise
         :returns: The new timeseries group.
         """
         all_attrs = dict(ts_attrs)
         all_attrs.update(data_attrs)
-        ts = kind(name=label, data=data, timestamps=times, **all_attrs)
+        if compress is True:
```

Comment:
Could just be `if compress:` (assuming we don't pass anything other than booleans!)